Hunting Queries/SecurityEvent/Least_Common_Process_Command_Lines.txt

```diff
@@ -0,0 +1,18 @@
+// Name: Least Common Processes Including Folder Depth
+// Id: 088d30e9-c02b-46b1-bd1f-d5b6d6b782f0
+// Description: Looks across your environment for least common Process Command Lines, may be noisy and require whitelisting.  By ZanCo
+// DataSource: #SecurityEvent
+// Tactics: #InitialAccess, #Execution, #Persistance
+let Whitelist = dynamic (['foo.exe', 'baz.exe']);
+let Sensitivity = 5;
+let StartDate = ago(7d);
+let Duration = 7d;
+SecurityEvent
+| where EventID == 4688 and TimeGenerated > StartDate and TimeGenerated < (StartDate + Duration)
```

Comment:
Probably should take conhost.exe out - | where NewProcessName !has "conhost.exe"

Reply:
I added conhost to the whitelist with foo.exe and baz.exe